spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods Index Page' do 
+  it 'shows the total number of items returned by the search' do 
+    visit '/'
+
+    fill_in 'q', with: "sweet potatoes"
+
+    click_button 'Search'
+    
+    expect(page).to have_content('Total hits for this search: 49676')
```

Comment:
We'd expect to see some more assertions here with the other attributes you display